README.md

```diff
@@ -97,9 +97,9 @@ Install the following plugins in your preferred editor
 
 * `yarn run build` to generate resource bundles and JS webpack bundles.
 * `yarn run watch` to only generate JS webpack bundles on file changes.
-* `yarn run test` launches karma tests with PhantomJS.
+* `yarn run test` launches jest tests with PhantomJS.
```

Comment:
Does jest also use phantom? I only see it being removed in configs/ package.json

Reply:
can remove it!